taxonomy-editor-frontend/src/pages/search/index.jsx

```diff
@@ -24,6 +24,7 @@ const SearchNode = ({ setDisplayedPages }) => {
       setDisplayedPages([
         { url: urlPrefix + "entry", translationKey: "Nodes" },
         { url: urlPrefix + "search", translationKey: "Search" },
+        { url: urlPrefix + "errors", translationKey: "Errors" },
```

Comment:
Waow, on each menu change we edit the same code every where.

It's for another PR, but shouldn't this menu be a component per se ? (used on every page)

Reply:
Yeah, seemed redundant to me too, I'll create an issue after this one.

Reply:
yep, I was planning to take that one